3_javascript/javascript1/week3/script3.js

```diff
@@ -0,0 +1,135 @@
+// Remove the item in names that is equal to nameToRemove
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+let remainingNames = names.filter(name => name !==nameToRemove);  
+  console.log(remainingNames); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+//   When will we be there??
+// Write a function where you speficy your speed in km/h and how far you have to go in km. 
+// The function has to return the time it will take to arrive at your destination. 
+// The time should be formatted like this: 
+// 3 hours and 34 minutes.
+
+function totalTravelTime (travelInformation) {
+  const {speed, destinationDistance} = travelInformation;
+  const totalHours = destinationDistance / speed;
+  const hours = Math.floor(totalHours);
+  const minutes = Math.round((totalHours - hours) * 60);
+
+  let calculatedTime = `${hours} hours`;
+
+  if (minutes > 0) {
+    calculatedTime += ` and ${minutes} minutes`;
+  }
```

Comment:
A small suggestion: think about how you might adjust the wording for cases with **1 hour** or **1 minute** to make it grammatically correct. 
It's these little touches that can elevate your solution. Keep up the great work! 😊